### ISSUE TITLE
Calls between non-focused funcs are not shown

### ISSUE THREAD

**Issue by TrueFurby** [TrueFurby/go-callvis#4]
Currently only calls to or from focused package are shown. However in cases when the results two contain functions which are both outside focused package and shown because focused package has some calls from/to both. Calls between such functions should be shown since both functions are already in the result anyway.

### Example code

```Go
package main

import (
    "fmt"

    "golang.org/x/sync/errgroup"
)

func main() {
    var g errgroup.Group
    g.Go(func() error {
        fmt.Println("Hello world")
        return nil
    })
    g.Wait()
}
```

### Result
![result](https://cloud.githubusercontent.com/assets/1229233/21941315/7d7e16ea-d9c8-11e6-95b4-9e6e022a3ea6.png)

> You can see that `main` calls `(*Group).Go` and `(*Group).Go$1` calls `main$1`. However [the call](https://github.com/golang/sync/blob/master/errgroup/errgroup.go#L58) between those two funcs inside `errgroup` is not shown even though both are in the result already.

By showing such calls the results would make clearer results by showing complete call path from the `main` to the `fmt.Println`.

Thanks @DmitriyMV for reporting this issue!

**Comment by DmitriyMV** [comment]
This is actually rather important, because there are libraries who are calling you back in the new goroutines. I'm looking forward to it 😃

**Comment by TrueFurby** [comment]
Fixed in [v0.3.4](https://github.com/TrueFurby/go-callvis/releases/tag/v0.3.4). 

Output for the above code looks like this now:
![already](https://cloud.githubusercontent.com/assets/1229233/22286159/822d8a2a-e2ee-11e6-9ab4-e0c5bae5b682.png)
